timestamp/fuzz_test.go

```diff
@@ -0,0 +1,11 @@
+package timestamp
+
+import (
+	"testing"
+)
+
+func FuzzParseSignedToken(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		_, _ = ParseSignedToken(data)
```

Comment:
Just an callout: As of now we are not performing timestamping so `ParseSignedToken` method is dead code. But we will be adding time-stamping support in future so it doesnt hurts to add a fuzzer.